src/cloudlabs/tasks/worker.py

```diff
@@ -29,21 +38,27 @@ def deploy(host_id):
         host.update(status=HostStatus.error,
                     deploy_log=host.deploy_log +
                     '\n\nTime limit exceeded - deployment terminated!\n')
+    except RuntimeError:  # if terminated -- to prevent worker being killed
+        pass
     except Exception as e:
         host.update(status=HostStatus.error,
                     deploy_log=host.deploy_log +
                     '\n\nUnexpected error!\n' + traceback.format_exc(e))
 
 
 @celery.task(name='cloudlabs.destroy')
-def destroy(host_id):
+def destroy(host_id, hard):
+    print("Destroying {} from task {}".format(host_id, deploy.request.id))
```

Comment:
This is probably a leftover debug message that should be a log message when we implement a logger for CloudLabs in a separate PR.